### PR TITLE
Made frontend changes to Ride Requests page in order to include the new pickup room and notes fields

### PR DESCRIPTION
**Pull request by pabucsb**
Please merge all of my previous PRs in the order they were created. Not pulling from main every time shows that more files where changed than they actually were on a given PR.

In this PR I made additions to our frontend code in order to give functionality to the new member variables I added in my first PR. 

Link to storybook: https://ucsb-cs156-m23.github.io/proj-gauchoride-m23-9am-1/

image

### CODE REVIEW

**Review by Aasish-Virjala — CHANGES_REQUESTED** [review]
Just a comment that needs to be removed, otherwise LGTM

File: frontend/src/main/components/Ride/RideForm.js [hunk 1]
```diff
@@ -182,6 +183,43 @@ function RideForm({ initialContents, submitAction, buttonLabel = "Create" }) {
...
+                <Form.Label htmlFor="notes">Notes</Form.Label>
+                <Form.Control
+                    data-testid={testIdPrefix + "-notes"}
+                    id="notes"
+                    type="text"
+                    {...register("notes")}
+                    isInvalid={Boolean(errors.notes)}
+                    //{...register("dropoffRoom", {
+                        //required: "Drop OffRoom number is required."
+                    //})}
```
Comment (pconrad) [inline frontend/src/main/components/Ride/RideForm.js:215]: Commented out code?
```suggestion
```

File: frontend/src/main/components/Ride/RideForm.js [hunk 2]
```diff
@@ -147,21 +147,22 @@ function RideForm({ initialContents, submitAction, buttonLabel = "Create" }) {
...
-                    {...register("room", {
-                        required: "Room number is required."
-                    })}
-                    placeholder="e.g. 2225"  
-                    defaultValue={initialContents?.room} 
+                    {...register("dropoffRoom")}
+                    isInvalid={Boolean(errors.dropoffRoom)}
+                    //{...register("dropoffRoom", {
+                        //required: "Drop OffRoom number is required."
+                    //})}
```
Comment (pconrad) [inline frontend/src/main/components/Ride/RideForm.js:159]: commented out code?

File: frontend/src/main/components/Ride/RideForm.js [hunk 3]
```diff
@@ -182,6 +183,40 @@ function RideForm({ initialContents, submitAction, buttonLabel = "Create" }) {
...
+                <Form.Label htmlFor="pickup-room">Pickup Room</Form.Label>
+                <Form.Control
+                    data-testid={testIdPrefix + "-pickup-room"}
+                    id="pickup-room"
+                    type="text"
+                    {...register("pickupRoom")}
+                    isInvalid={Boolean(errors.pickupRoom)}
+                    //{...register("dropoffRoom", {
+                        //required: "Drop OffRoom number is required."
+                    //})}
```
Comment (pconrad) [inline frontend/src/main/components/Ride/RideForm.js:196]: commented out code?

File: frontend/src/main/components/Ride/RideForm.js [hunk 4]
```diff
@@ -180,6 +180,41 @@ function RideForm({ initialContents, submitAction, buttonLabel = "Create" }) {
...
+            <Form.Group className="mb-3" >
+                <Form.Label htmlFor="pickup-room">Pickup Room</Form.Label>
+                <Form.Control
+                    data-testid={testIdPrefix + "-pickup-room"}
+                    id="pickup-room"
+                    type="text"
+                    {...register("pickupRoom")}
+                    isInvalid={Boolean(errors.pickupRoom)}
+                    //{...register("dropoffRoom", {
+                        //required: "Drop OffRoom number is required."
```
Comment (pabucsb) [inline frontend/src/main/components/Ride/RideForm.js:192]:
```suggestion
                        
```

**Review by pconrad — CHANGES_REQUESTED** [review]
Once again, asking about commented out code.

**Review by pconrad — APPROVED** [review]
LGTM

**Pull request merged**